src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -355,6 +364,27 @@ export default class ExportImportService
         await this.importDependencies(dto, user, auditUser);
     }
 
+    private async importLinks(dto: ImportTogglesSchema, auditUser: IAuditUser) {
```

Comment:
similar algorithm to feature tags. First delete previous links then create new ones using existing method for creation to get all event logs recorded